app/plugin.go

```diff
@@ -497,7 +497,7 @@ func (a *App) getRemoteMarketplacePlugin(pluginId, version string) (*model.BaseM
 	return plugin, nil
 }
 
-func (a *App) getRemotePlugins(filter *model.MarketplacePluginFilter) (map[string]*model.MarketplacePlugin, *model.AppError) {
+func (a *App) getRemotePlugins(_ *model.MarketplacePluginFilter) (map[string]*model.MarketplacePlugin, *model.AppError) {
```

Comment:
I'm not sure why `*model.MarketplacePluginFilter` is passed at all, but I just omitted the name to make clear it's not used.

Reply:
Probably a relic from when we were going to pass through filtering settings upstream. I'm fine with removing the parameter altogether.

Reply:
Removed it :+1: